examples/node/src/main.ts

```diff
@@ -12,7 +14,7 @@ for (const key in wrtc) {
 async function connect() {
   const host = process.env.HOST;
   const apiKeyId = process.env.API_KEY_ID;
-  const apiKeySecret = process.env.API_KEY_SECRET;
+  const apiKeySecret = process.env.API_KEY;
```

Comment:
The README uses `API_KEY` in the `.env` file instead of `API_KEY_SECRET` and this naming is more consistent across the other SDK documentation.